tests/pytest/core/test_admin.py

```diff
@@ -0,0 +1,8 @@
+import pytest
+
+
+@pytest.mark.django_db
+def test_admin_registered(client):
+    response = client.get("/admin")
+
+    assert response.status_code == 301
```

Comment:
Could add this test I wrote for the `pre_login_user` method: https://github.com/cal-itp/benefits/pull/1855/commits/0d5984ef1b2c47321b13a4f0709989795c501723 Feel free to refactor as you'd like 😅

Reply:
Added slightly modified versions of this test in eeffe2ec43b5af53ae2a4f4870731dbe99579be0